model/annotations.go

```diff
@@ -1044,7 +1044,23 @@ func (r *PdfReader) newPdfAnnotationTextFromDict(d *core.PdfObjectDictionary) (*
 func (r *PdfReader) newPdfAnnotationLinkFromDict(d *core.PdfObjectDictionary) (*PdfAnnotationLink, error) {
 	annot := PdfAnnotationLink{}
 
-	annot.A = d.Get("A")
+	if obj := d.Get("A"); obj != nil {
```

Comment:
Might be better to use `core.GetIndirect` here in order to resolve references as well:

```
if obj := d.Get("A"); obj != nil {
	if indObj, isIndirect := core.GetIndirect(obj); isIndirect {
		actionObj, err := r.newPdfActionFromIndirectObject(indObj)
		if err != nil {
			return nil, err
		}
		annot.A = actionObj
	} else if !core.IsNullObject(obj) {
		return nil, errors.New("A should point to an indirect object")
	}
}
```